catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -124,10 +124,36 @@ public List<String> listTables(String databaseName) throws NoSuchSchemaException
     }
   }
 
+  /**
+   * Get table information from the result set and attach it to the table builder, If the table is
+   * not found, it will throw a NoSuchTableException.
+   *
+   * @param resultSet The result set of the table
+   * @return The builder of the table to be returned
+   */
+  protected JdbcTable.Builder getTableBuilder(
+      ResultSet resultSet, String databaseName, String tableName) throws SQLException {
+    boolean found = false;
+    // Handle case-sensitive issues.
```

Comment:
What does this comment refer to? I don't see any code below that handles case sensitivity.

Reply:
It's the code `Objects.equals(resultSet.getString("TABLE_NAME"), tableName)`, Previously,  we would get `T` or `t` if we use keyword `t` to query table.

Reply:
It's not part of this PR, it was added previously.

Reply:
The comment should be clearer.

Reply:
> The comment should be clearer.

added.